storage/storage.c

```diff
@@ -88,22 +88,11 @@ const uint32_t V0_PIN_EMPTY = 1;
 // The total number of iterations to use in PBKDF2.
 #define PIN_ITER_COUNT 20000
 
-// The number of milliseconds required to execute PBKDF2.
-#define PIN_PBKDF2_MS 1280
+// The number of CPU cycles required to execute PBKDF2.
+#define PIN_PBKDF2_CYCLES 222000000
```

Comment:
It certainly would, but the question is how much difference do these things make, considering that all models use very similar architecture, compiler and optimizations. They mainly differ in the clock speed. When I made some measurements across different models, the execution time was more or less inversely proportional to the clock speed. It certainly seems more accurate to use cycles as the base point for the time estimate, rather than milliseconds.